frontend/workflows/projectSelector/src/dash-hooks.tsx

```diff
@@ -12,6 +16,10 @@ type useDashUpdaterReturn = {
   updateSelected: (state: DashState) => void;
 };
 
+type useTimelineUpdaterReturn = {
+  updateTimeline: (key: string, points: IClutch.timeseries.v1.IPoint[]) => void;
```

Comment:
instead of `key: string, points: IClutch.timeseries.v1.IPoint[]`, should we reuse the interface type already defined?